draft-kazuho-httpbis-priority.md

```diff
@@ -349,6 +349,40 @@ server that receives requests for a font {{?RFC8081}} and images with the same
 urgency might give higher precedence to the font, so that a visual client can
 render textual information at an early moment.
 
+## Reprioritization
+
+Once a client sends a request, it cannot reprioritize the corresponding response
+by using the Priority header field.  This is because an HTTP header field can
+only be sent as part of an HTTP message.
+
+Therefore, to support reprioritization, it is necessary to define a
+HTTP-version-dependent mechanism for transmitting the priority parameters.
+
+One approach that we can use in HTTP/2 ({{?RFC7540}}) is to use a frame that
+carries the priority parameters.
+
+~~~ drawing
+  0                   1                   2                   3
+  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ +---------------------------------------------------------------+
+ |R|                        Stream ID (31)                       |
+ +---------------------------------------------------------------+
+ |                   Priority Field Value (*)                  ...
+ +---------------------------------------------------------------+
+~~~
+{: #fig-cert-needed title="Repriotizization frame payload"}
```

Comment:
nit #fig-cert-needed but this doesn't matter since it is not referenced in the document

Reply:
lol. Nice catch!